app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -416,8 +415,14 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler, AppStateLis
         }
     }
 
+    private fun onAppUpdated() {
+        showWhatsNew()
+        settingsHandler.showWhatsNew = false
+    }
+
     private fun showWhatsNew() {
-        navController.navigate(R.id.whatsNewDialog)
+        //TODO: uncomment if whats new screen should be shown
+        //navController.navigate(R.id.whatsNewDialog)
```

Comment:
Why do we need to do this manually?

Reply:
we do not have version based logic yet. What's new is always shown for the new version.